stix_shifter_utils/stix_transmission/utils/RestApiClientAsync.py

```diff
@@ -57,6 +57,17 @@ def __init__(self, host, port=None, headers={}, url_modifier_function=None, cert
         unique_file_handle = uuid.uuid4()
         self.server_cert_name = "/tmp/{0}-server_cert.pem".format(unique_file_handle)
         server_ip = host
+        
+        #To enable proxy, set the environment variable "STIX_SHIFTER_ENABLE_TRUST_ENV" to true. This option will allow the connection
+        #to use the system environments proxy settings. This can be done by setting the "https_proxy" environment variable to 
+        #"http(s)://[username]:[password]@[hostname]/[ipaddress]:[port]". Alternative proxy schema's may or may not work.
+        self.trust_env_enabled = os.environ.get("STIX_SHIFTER_ENABLE_TRUST_ENV", False).lower()
```

Comment:
Do you think STIX_SHIFTER_ENABLE_TRUST_PROXY_ENV would be more descriptive or just leave it as it is ?

Reply:
I think we need to remove the .lower there.